probe-rs/src/probe/common.rs

```diff
@@ -215,7 +215,7 @@ pub(crate) fn extract_ir_lengths(
         // If the number of possible starts matches the number of TAPs,
         // we can unambiguously find all lengths.
         let irlens = starts_to_lengths(&starts, ir.len());
-        tracing::info!("IR lengths are unambiguous: {irlens:?}");
+        tracing::info!("IR lengths are ambiguous: {irlens:?}");
```

Comment:
Did you mean to change this?

Reply:
uh, I guess I might have misunderstood this part :) bear with me, work in progress
thanks for the note